### PR TITLE
update-feature2-paragraph

### PR DESCRIPTION
**Pull request by KrenCon**

### CODE REVIEW

File: src/feature-2/feature2.1.txt [hunk 1]
```diff
@@ -1,2 +1,10 @@
-Create a new feature 2.1
- 
\ No newline at end of file
+Edited the feature 2.1
```
Comment (haqhuu) [inline src/feature-2/feature2.1.txt:1]: delete this line

File: src/feature-2/feature2.1.txt [hunk 2]
```diff
@@ -1,2 +1,10 @@
...
+
+
+Feature is create a product;
```
Comment (haqhuu) [inline src/feature-2/feature2.1.txt:4]:
```suggestion
Feature is create a cart page;
```

File: src/feature-2/feature2.1.txt (hunk 1 above)
Comment (haqhuu) [inline src/feature-2/feature2.1.txt:1]: acb

File: src/feature-2/feature2.1.txt (hunk 2 above)
Comment (haqhuu) [inline src/feature-2/feature2.1.txt:4]:
```suggestion
Feature is create carrrt
```

File: src/feature-2/feature2.1.txt (hunk 1 above)
Comment (KrenCon) [reply src/feature-2/feature2.1.txt:1]: ddsds

**Pull request merged**